modules/importing-repositories/procedure-understanding-bulk-import-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+= Bulk Import limitations
+
+* Only the first 20 repositories (in alphabetical order) can be displayed at most on the Added Repositories page.
+Also, the count of Added Repositories displayed might be wrong.
+In future releases, we plan to address this with proper pagination.
```

Comment:
Is this information necessary for the users to know, what if the plan changes? I suggest we remove it
```suggestion
In future releases, this issue can be addressed with proper pagination.
```